### PR TITLE
Issue-562 | Added code changes to update pod scheduling error message…

### PR DESCRIPTION
**Pull request by Rahul1804**
Details about code changes

https://github.com/bluek8s/kubedirector/issues/562

Testing

Created 5 member ubuntu cluster

Screenshot 2022-03-18 at 12 12 52 AM

Couple of pods' scheduling failed because of less resource availability in the k8s cluster. 

Screenshot 2022-03-17 at 11 56 04 PM

Then untainted the master node so that some of the pods can be scheduled on master.

Screenshot 2022-03-17 at 11 57 16 PM

Change in kd node id 5 status - Configured And No Scheduling Error Message

Screenshot 2022-03-18 at 12 00 45 AM

### CODE REVIEW

**Comment by joel-bluedata** [comment]
Plz add some details here about what you tested and an example of what the relevant status fields look like in one of the test cases.

File: pkg/controller/kubedirectorcluster/cluster.go [hunk 1]
```diff
@@ -619,6 +623,9 @@ func updateStateRollup(
 				// Count missing container as waiting, at this point.
 				if memberStatus.StateDetail.LastKnownContainerState == containerMissing {
 					cr.Status.MemberStateRollup.MembersWaiting = true
+					if memberStatus.StateDetail.SchedulingErrorMessage != nil {
+						cr.Status.MemberStateRollup.MembersNotScheduled = true
```
Comment (neogeographica) [inline pkg/controller/kubedirectorcluster/cluster.go:627]: In the block of stuff at the beginning of this function we must also set MembersNotScheduled to false initially.

File: pkg/controller/kubedirectorcluster/util.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 Hewlett Packard Enterprise Development LP
```
Comment (neogeographica) [inline pkg/controller/kubedirectorcluster/util.go:1]: Copyright 2022!

**Comment by neogeographica** [comment]
Just a couple of minor fixes noted above, then this will be ready to merge.

**Comment by neogeographica** [comment]
Sorry I'm logged into my personal GitHub account rather than corporate. Lemme switch...

**Comment by joel-bluedata** [comment]
OK this should make it more official. :-) Just a couple of minor fixes noted above, then this will be ready to merge.

**Review by joel-bluedata — APPROVED** [review]

**Pull request merged**